backend/src/main/resources/application.properties

```diff
@@ -16,5 +16,7 @@ server.port=8010
 
 #Needed due to empty return msg from json
 server.error.include-message=always
+
 #JWT
 jwt.jwtExpirationMs=3600000
+jwt.jwtSecret=9z$C&F)J@McQfTjWnZr4u7x!A%D*G-Ka
```

Comment:
ditto in prev review (two times the change in code)

Reply:
https://github.com/Inzynierowie/Inzynierka/pull/51#discussion_r534212598

Reply:
an same here 
https://github.com/Inzynierowie/Inzynierka/pull/51#discussion_r537013442

Reply:
@Lukortech Same here like in https://github.com/Inzynierowie/Inzynierka/pull/51#issuecomment-739525779 ?

Reply:
Will be changed with #52